_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -6016,8 +6041,22 @@ void MfxHwH264Encode::SetDefaults(
         extDdi->CabacInitIdcPlus1 = GetCabacInitIdc(par.mfx.TargetUsage) + 1;
     }
 
-    if (hwCaps.UserMaxFrameSizeSupport && extOpt2->MaxFrameSize == 0)
-        extOpt2->MaxFrameSize = MFX_MIN(GetMaxFrameSize(par), GetFirstMaxFrameSize(par));
+    bool IsEnabledSwBrc = false;
+#if !defined(MFX_EXT_BRC_DISABLE)
```

Comment:
```
[dvrogozh@dvrscl MediaSDK]$ fgrep -rsn MFX_EXT_BRC_DISABLE
[dvrogozh@dvrscl MediaSDK]$
```

Can we, please, avoid to introduce macro definitions which we do not use anywhere in the project?